backend/src/test/resources/application.properties

```
@@ -0,0 +1 @@
+spring.mongodb.embedded.version=3.5.5
```

Implement your database maybe?

```suggestion
spring.mongodb.embedded.version=3.5.5
spring.data.mongodb.database=YouLearnersDB
```